spec/request/documents_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe 'Documents management', type: :request do
+  it 'should fail if no file is provided' do
+    post '/v0/claims/1/documents'
+    expect(response).to_not be_success
```

Comment:
I couldn't get a controller test to work. It said something about the route not being available? I guess it had something to do with documents being a sub-resource instead of top level, but I didn't figure it out.